src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, OverlayTrigger, Tooltip } from '@edx/paragon';
+import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
+
+const AssignmentRowActionTableCell = ({ row }) => (
+  <div className="d-flex">
+    {row.original.state === 'allocated' && (
+      <OverlayTrigger
+        key="Remind"
+        placement="top"
+        overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
+      >
+        <Icon
+          className="ml-auto mr-0"
+          src={Mail}
+          // eslint-disable-next-line no-console
+          onClick={() => console.log(`Reminding ${row.original.uuid}`)}
+          data-testid={`remind-assignment-${row.original.uuid}`}
+        />
+      </OverlayTrigger>
+    )}
+    <div />
```

Comment:
[nit/question] Is this empty `div` just being used for spacing between these two icons? If so, I would recommend wrapping these icons with `<Stack direction="horizontal" gap={2}>` instead of relying on an empty div for styling.